src/main/java/com/gregtechceu/gtceu/data/recipe/configurable/RecipeAddition.java

```diff
@@ -988,11 +1030,321 @@ private static void hardMiscRecipes(Consumer<FinishedRecipe> provider) {
+                    .duration(80).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "loom", new ItemStack(Blocks.LOOM), "SSS",
+                    "PdP", "XYX",
+                    'S', new ItemStack(Items.STRING),
+                    'P', new UnificationEntry(plate, Wood),
+                    'X', new UnificationEntry(screw, Wood),
+                    'Y', ItemTags.PLANKS);
+
+            ASSEMBLER_RECIPES.recipeBuilder("loom")
+                    .inputItems(new ItemStack(Items.STRING, 3))
+                    .inputItems(plate, Wood, 2)
+                    .inputItems(ItemTags.PLANKS)
+                    .outputItems(new ItemStack(Blocks.LOOM))
+                    .duration(80).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "smoker", new ItemStack(Blocks.SMOKER), "wId",
+                    "PFP", "XPX",
+                    'I', new ItemStack(Blocks.IRON_BARS),
+                    'P', new UnificationEntry(plate, TreatedWood),
+                    'X', new UnificationEntry(bolt, Iron),
+                    'F', new ItemStack(Blocks.FURNACE));
+
+            ASSEMBLER_RECIPES.recipeBuilder("smoker")
+                    .inputItems(new ItemStack(Blocks.IRON_BARS))
+                    .inputItems(ItemTags.PLANKS, 3)
+                    .inputItems(new ItemStack(Blocks.FURNACE))
+                    .outputItems(new ItemStack(Blocks.SMOKER))
+                    .duration(80).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "blast_furnace", new ItemStack(Blocks.BLAST_FURNACE), "wfd",
+                    "PFP", "XPX",
+                    'P', new UnificationEntry(plate, Iron),
+                    'X', new UnificationEntry(screw, Iron),
+                    'F', new ItemStack(Blocks.FURNACE));
+
+            ASSEMBLER_RECIPES.recipeBuilder("blast_furnace")
+                    .inputItems(plate, Iron, 3)
+                    .inputItems(new ItemStack(Blocks.FURNACE))
+                    .outputItems(new ItemStack(Blocks.BLAST_FURNACE))
+                    .duration(80).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "composter", new ItemStack(Blocks.COMPOSTER), "PsP",
+                    "P P", "XPX",
+                    'P', ItemTags.PLANKS,
+                    'X', new UnificationEntry(screw, Wood));
+
+            ASSEMBLER_RECIPES.recipeBuilder("composter")
+                    .inputItems(ItemTags.PLANKS, 4)
+                    .circuitMeta(1)
+                    .outputItems(new ItemStack(Blocks.COMPOSTER))
+                    .duration(80).EUt(6).save(provider);
+
+            ASSEMBLER_RECIPES.recipeBuilder("bell")
+                    .inputItems(new ItemStack(Items.STICK))
+                    .inputItems(rod, Stone, 2)
+                    .inputItems(bolt, Iron, 2)
+                    .inputItems(plate, Bronze, 4)
```

Comment:
I'd use brass instead of bronze for the bell

Reply:
I used brass originally but apparently bells are made of bronze

Reply:
Lmk if I should still change it

Reply:
oh, I see.